Applications/MIDIMonitor/English.lproj/docs.htmld/index.html

```diff
@@ -35,6 +35,7 @@
             <li><a href="#howto">How to use it</a></li>
             <li><a href="#preferences">Preferences</a></li>
             <li><a href="#advanced">Advanced</a></li>
+            <li><a href="#uninstall">Uninstall</a></li>
```

Comment:
Let's please do this in a separate PR, since it's unrelated to the other changes.

Reply:
Okay. Can I undo this just using a revert to the last commit? Or is there a better way?

Reply:
Yes, I think that should work.